include/EUTelGeometryTelescopeGeoDescription.tcc

```diff
@@ -0,0 +1,50 @@
+#ifndef EUTELGEOMETRYTELECOPEGEODESCRIPTION_TCC  
+#define EUTELGEOMETRYTELECOPEGEODESCRIPTION_TCC
+
+/** Determine id of the sensor in which point is locate
+ *  * 
+ *  * @param globalPos 3D point in global reference frame
+ *  * @return sensorID or -999 if the point in outside of sensor volume
+ *  */
+//MUST OUTPUT -999 TO SIGNIFY THAT NO SENSOR HAS BEEN FOUND. SINCE USED IN PATTERN RECOGNITION THIS WAY.
```

Comment:
No, please pass the found sensorID via a reference and make the return type boolean and return true if sensor is found.
Instead of writing:

int foundSensorID = getSensorID( ... );
if(foundSensorID != -999)
{ //do stuff }

write:
int foundSensorID;
if(getSensorID( ..., foundSensorID ))
{ //do stuff }